shell/platform/windows/flutter_windows_view.cc

```diff
@@ -661,28 +662,24 @@ FlutterWindowsEngine* FlutterWindowsView::GetEngine() {
 }
 
 void FlutterWindowsView::AnnounceAlert(const std::wstring& text) {
-  AccessibilityRootNode* root_node =
-      binding_handler_->GetAccessibilityRootNode();
-  AccessibilityAlert* alert =
-      binding_handler_->GetAccessibilityRootNode()->GetOrCreateAlert();
-  alert->SetText(text);
-  HWND hwnd = GetPlatformWindow();
-  NotifyWinEventWrapper(EVENT_SYSTEM_ALERT, hwnd, OBJID_CLIENT,
-                        AccessibilityRootNode::kAlertChildId);
-}
-
-void FlutterWindowsView::NotifyWinEventWrapper(DWORD event,
-                                               HWND hwnd,
-                                               LONG idObject,
-                                               LONG idChild) {
-  if (hwnd) {
-    NotifyWinEvent(EVENT_SYSTEM_ALERT, hwnd, OBJID_CLIENT,
-                   AccessibilityRootNode::kAlertChildId);
+  binding_handler_->Alert(text);
+  ui::AXPlatformNodeWin* alert_node = binding_handler_->GetAlert();
+  NotifyWinEventWrapper(alert_node, ax::mojom::Event::kAlert);
```

Comment:
Why doesn't `WindowBindingHandler::Alert` call `AXPlatformNodeWin::NotifyAccessibilityEvent`?

Reply:
For mocking in the unit tests.

Reply:
Taking a step back, the value of the `FlutterWindowsView` / `FlutterWindow` split is to enable mock testing by decoupling Flutter from native win32 APIs. We can't mock win32 APIs, so instead we wrap them in `FlutterWindow` and mock that.

Ideally `FlutterWindow` is as small as possible. Any code that's unit testable should be moved out of `FlutterWindow` and into `FlutterWindowsView`.

Currently, the alerting logic is split across `FlutterWindow` and `FlutterWindowsView`. Do you think we could refactor `FlutterWindow` to push up more alerting logic to `FlutterWindowsView`?